system-addon/lib/ActivityStream.jsm

```diff
@@ -81,6 +82,20 @@ const PREFS_CONFIG = new Map([
       "provider_name": "Pocket",
       "provider_icon": "pocket"
     }`
+  }],
+  ["migrationExpired", {
+    title: "Boolean flag that decides whether to show the migration message or not.",
+    value: false,
+    value_local_dev: false
```

Comment:
The other review comment asked about why `value_local_dev` is needed. You addressed the last one but left these untouched.